modules/images/webp-uploads/load.php

```diff
@@ -8,6 +8,13 @@
  * @package performance-lab
  */
 
+// Define the constant.
+if ( defined( 'WEBP_UPLOADS_VERSION' ) ) {
+	return;
+}
+
+define( 'WEBP_UPLOADS_VERSION', '1.0.0' );
```

Comment:
I think we should set this to a version for the module, since it's in the module file, not the plugin:
```suggestion
define( 'WEBP_UPLOADS_VERSION', 'Performance Lab ' . PERFLAB_VERSION );
```

What that means though is that we should replace the version in this constant with the standalone plugin version as part of the build script. Can you open a separate follow up PR to do that? Let's not block this one on this new requirement.

Basically the new requirement for the separate PR should be:
* For any standalone plugin, the version constant name that it uses should be added to the `plugins.json` definition.
* The `build-plugins` command should be enhanced to replace the version set in the constant with the current plugin version, e.g. `WEBP_UPLOADS_VERSION` value should change from `Performance Lab 2.1.0` to `1.0.0`.

Reply:
I'm a big fan of trying to automatically set these constants via the `plugins.json` file so we're handling all configuration in one place. We could probably also automatically fill out the array in `perflab_get_standalone_plugins_config` using the same JSON file so the constant is always based on the slug of each plugin from that file.

Reply:
Thanks @felixarntz, I open #681 as a follow-up PR that updates the constant value in `build-plugins` command.